### PR TITLE
Add the InputDeviceCapabilities IDL file with test

### PR DESCRIPTION
**Pull request by lukebjerring**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-03-02 20:01:23
Finished: 2018-03-02 20:11:04


<h2>Failing Jobs</h2>
<ul>

<li>chrome:dev</li>

<li>safari:11.0</li>

<li>MicrosoftEdge:16.16299</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25159)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/348381192)

**Comment by foolip** [comment]
This is odd. I found http://wicg.github.io/InputDeviceCapabilities/tests/ outside of wpt. @RByers, are those good tests that should be moved?

**Comment by foolip** [comment]
@RByers is there someone who now works on those features who might be able to move those tests? Especially ones for bits we've already shipped would be valuable, and if they reveal missing testdriver bits I'm sure @kereliuk would like to know.

**Comment by foolip** [comment]
@SummerLW can you take a look? Are you OK with being in the OWNERS so that you're added on new reviews in this directory?

**Comment by NavidZ** [comment]
We should be able to do that. As soon as @kereliuk lands his CL for pointer actions in wpt we should be able to start updating those manual tests and add the automation instructions.

File: interfaces/InputDeviceCapabilities.idl [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+[Constructor(optional InputDeviceCapabilitiesInit deviceInitDict)]
+interface InputDeviceCapabilities {
+    readonly attribute boolean firesTouchEvents;
+    readonly attribute boolean pointerMovementScrolls;
+};
+
+dictionary InputDeviceCapabilitiesInit {
+    boolean firesTouchEvents = false;
+    boolean pointerMovementScrolls = false;
```
Comment (NavidZ) [inline interfaces/InputDeviceCapabilities.idl]: Even Chrome hasn't implemented this bit :). But I guess from the test point of view this would be alright.

**Comment by SummerLW** [comment]
Yes, I will take a look. Of course, I would like to keep track of them.

On Fri, Jun 8, 2018 at 11:28 AM, Philip Jägenstedt <notifications@github.com

**Review by NavidZ — APPROVED** [review]
This particular API has been in Chrome for quite a while and not been implemented by other browsers. I'll try to come up with a plan say next quarter to get them more engaged hopefully.

File: interfaces/InputDeviceCapabilities.idl (hunk 1 above)
Comment (foolip) [reply interfaces/InputDeviceCapabilities.idl]: Yep, I noticed it was failing. But correctly so, should be removed from spec if it stays failing in all browsers forever, so having the test can help surface that later :)

**Comment by foolip** [comment]
Thanks @SummerLW @NavidZ, I'll go ahead and merge this.

**Pull request merged**